src/RoleObjects/AccessibilityObject.js

```diff
@@ -377,7 +406,8 @@ export default class AccessibilityObject {
   /**
    * Sets whether the DisplayObject is picked up for a drag and drop interaction
    * @access public
-   * @param {boolean} dragging - true if being dragged, false for not being dragged but supports it, undefined for not being dragged and doesn't support it
+   * @param {boolean} dragging - true if being dragged, false for not being dragged but
+    supports it, undefined for not being dragged and doesn't support it
```

Comment:
Nit: see above multiline function comment issue.  I'm going to stop copy and pasting this now.  It's applicable to at least this whole file if not PR.